aldryn_client/client.py

```diff
@@ -585,11 +585,14 @@ def init_docker(self, sitename, path=None):
             raise e
 
         # build web container
-        self.docker_client.build(
+        build_result = self.docker_client.build(
             path=site_path,
             tag='myweb',
             rm=True,
         )
+        # Consume the generator returned by build() - for some versions
+        # of docker / docker_py this is the only way to make it work.
+        _ = list(build_result)
```

Comment:
Nope, you have to catch StopIteration only if you call `next` yourself; by iterating with:

```
for _ in build_result:
    pass
```

The `StopIteration` exception is what terminates the for loop.